src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,382 @@
+
+1. Add a sidecar container to the pod containing the New Relic Kubernetes Integration.
+2. If a secret doesn't exist, create one in the same namespace as the pod containing the New Relic license key, which is needed for the integration to report data.
+3. Add the pod’s service account to a `ClusterRoleBinding` previously created by the operator chart, which will grant this sidecar the required permissions to hit the Kubernetes metrics endpoints.
+
+The `ClusterRoleBinding` grants the following permissions to the pod being injected:
+
+```
+- apiGroups: [""]
+  resources:
+	- "nodes"
+	- "nodes/metrics"
+	- "nodes/stats"
+	- "nodes/proxy"
+	- "pods"
+	- "services"
+  verbs: ["get", "list"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+```
+
+<Callout variant="tip">
+Only pods created after the deployment of the monitoring solution will be injected with the configuration and agent. 
+</Callout>
+
+In order to get the sidecar injected on pods deployed before the operator has been installed, the user needs to manually perform a rollout (restart). New Relic has chosen not to do this automatically in order to prevent unexpected service disruptions and resource usage spikes.
+
+Here's the injection workflow:
+
+![Diagram showing the workflow of sidecar injection](./images/fargate-workflow.svg "Diagram showing the workflow of sidecar injection")
+
+#### Automatic injection installation [#auto-injection-install]
+
+<Callout variant="tip">
+These setup steps are for a generic installation. Before completing these, take a look at the [Configuration](#config-auto) section below to see if you want to modify any aspects of the automatic injection.
+</Callout>
+
+In order to install the operator in charge of injecting the infrastructure sidecar, all you have to do is add the following flags to the the [install bundle command](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#install-k8-helm):
+
+```
+helm upgrade --install newrelic newrelic/nri-bundle \
+--set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
+--set global.cluster=K8S_CLUSTER_NAME \
+--set infrastructure.enabled=true \
+--set prometheus.enabled=true \
+--set webhook.enabled=true \
+--set ksm.enabled=false \
+--set kubeEvents.enabled=true \
+<var>--set global.fargate=true</var> \
+<var>--set newrelic-infra-operator.enabled=true</var>
+```
+
+<Callout variant="important">
+In the case of a hybrid cluster (EC2 and Fargate), make sure after deploying that the solution is not selected by any Fargate profiles; otherwise, the `DaemonSet` instances will be stuck in a pending state. Fargate-only environments don't have this limitation because no `DaemonSet` instances are created.
```

Comment:
```suggestion
When deploying the solution on a hybrid cluster (with both EC2 and Fargate nodes), please make that the solution is not selected by any Fargate profiles; otherwise, the `DaemonSet` instances will be stuck in a pending state. For fargate-only environments this is not a concern because no `DaemonSet` instances are created.
```